### PR TITLE
Make a polymer-editor-service binary, for sublime text and others.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

---
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-editor-service/75)

### CODE REVIEW

File: bin/polymer-editor-service.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+// NOTE 04-21-2017: Confirmed "semver" supports Node versions as low as 0.10
+var semver = require('semver');
+
+var minVersion = '6';
+
+// Exit early if the user's node version is too low.
+if (!semver.satisfies(process.version, '>=' + minVersion)) {
+  console.log(
+      'Polymer CLI requires at least Node v' + minVersion + '. ' +
```
Comment (TimvdLippe) [inline bin/polymer-editor-service.js]: Copy pasta
image

Comment (rictic) [reply bin/polymer-editor-service.js]: Oops.

screen shot 2017-11-20 at 2 32 46 pm

File: bin/polymer-editor-service.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/**
+ * @license
+ * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```
Comment (valdrinkoshi) [inline bin/polymer-editor-service.js]: Should this be `Copyright (c) 2017`?

Comment (rictic) [reply bin/polymer-editor-service.js]: +1, yep

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <img src="https://user-images.githubusercontent.com/1659/32983803-946aabaa-cc4f-11e7-90ca-a63e8c437037.gif" width="821" height="588">
 
 - Added support for the command `polymer-ide/applyAllFixes`. This is a zero argument command that calls `workspace/applyEdit` with a `WorkspaceEdit` that applies all non-overlapping fixes for all fixable warnings in the package.
+- Warnings should now immediately be updated in response to any change, whether it happens in the editor or from external tools, like `bower install`.
```
Comment (valdrinkoshi) [inline CHANGELOG.md]: ups, seems this is a dupe of line 26

**Pull request merged**